src/qml/controls/OptionButton.qml

```diff
@@ -80,6 +81,28 @@ Button {
                     }
                 }
             }
+            Loader {
+                Layout.topMargin: 12
+                Layout.fillWidth: true
+                active: button.customDir.length > 0
+                visible: active
+                sourceComponent: Button {
+                    id: container
+                    background: Rectangle {
+                        color: Theme.color.neutral2
+                        radius: 5
+                    }
+                    font.family: "Inter"
+                    font.styleName: "Semi Bold"
+                    font.pixelSize: (Qt.platform.os === "android") ? Math.max(12, Qt.screenDpi / 100) : 13
```

Comment:
What's the reason for having this different for android?  Android also scales the fonts based on the dpi already so I'm not sure I understand the dpi factor here

Reply:
good call folks... done